### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by EPashkin**
cc @GuillaumeGomez

### CODE REVIEW

File: src/release.py [hunk 1]
```diff
@@ -167,14 +167,14 @@ def update_repo_version(repo_name, crate_name, crate_dir_path, temp_dir, update_
                 continue
             new_version = None
             if badges_only is False:
-                new_version = update_version(new_version, update_type, section.name)
+                new_version = update_version(version, update_type, section.name)
```
Comment (EPashkin) [inline src/release.py]: Without this I get
```
  File "src/release.py", line 61, in update_version
    version_split = version.replace('"', '').split('.')
AttributeError: 'NoneType' object has no attribute 'replace'
```

File: src/release.py [hunk 2]
```diff
@@ -464,8 +464,7 @@ def generate_new_tag(repository, temp_dir, specified_crate):
             versions[crate['crate']] = CRATES_VERSION[crate['crate']]
             if crate['crate'].endswith('-sys') or crate['crate'].endswith('-sys-rs'):
                 version = CRATES_VERSION[crate['crate']]
-    if specified_crate is not None and
-           (specified_crate.endswith('-sys') or specified_crate.endswith('-sys-rs')):
+    if specified_crate is not None and (specified_crate.endswith('-sys') or specified_crate.endswith('-sys-rs')):
```
Comment (EPashkin) [inline src/release.py]: This not was work for me, maybe something with my Python 3.6.4 wrong.

File: src/release.py [hunk 3]
```diff
@@ -143,7 +143,7 @@ def update_crate_version(repo_name, crate_name, crate_dir_path, temp_dir, specif
                 if find_crate(entry['key']):
                     section.set(entry['key'], CRATES_VERSION[entry['key']])
     result = write_into_file(file, "{}\n".format(toml))
-    write_msg('=> {}: {}'.format(output.split('/')[-2],
+    write_msg('=> {}: {}'.format(output.split(os_sep)[-2],
```
Comment (EPashkin) [inline src/release.py]: Just portability fix

File: src/release.py (hunk 2 above)
Comment (GuillaumeGomez) [reply src/release.py]: Please add parents around instead please. The line is just too long like this. :)

File: src/release.py (hunk 3 above)
Comment (GuillaumeGomez) [reply src/release.py]: Oh good catch!

File: src/release.py (hunk 1 above)
Comment (GuillaumeGomez) [reply src/release.py]: Thanks!

**Comment by GuillaumeGomez** [comment]
Just one nit and it's good. Thanks!

File: src/release.py (hunk 2 above)
Comment (EPashkin) [reply src/release.py]: You meant this way?

Comment (GuillaumeGomez) [reply src/release.py]: I mean like this:

```python
if (specified_crate is not None and
    (specified_crate.endswith('-sys') or specified_crate.endswith('-sys-rs'))):
```

Comment (EPashkin) [reply src/release.py]: You right, that looks better.
Done

Comment (GuillaumeGomez) [reply src/release.py]: Python's magic! :p

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**